### ISSUE TITLE
Drag and Drop Dependencies

### ISSUE THREAD

**Issue by mariohmol** [jsGanttImproved/jsgantt-improved#117]
Use the dot at the end of the arrow (that connects two tasks) as a draggable and the task as drop areas.

**Comment by mafster** [comment]
Just to clarify... jsGantImproved doesnt do this yet? are we able to adjust the start and end with any type of click and drag?

**Comment by mariohmol** [comment]
it doesnt.. it can be accomplish like:

1. make the point of the arrow dragglable
2. make tasks dropable
3. when drop, change the data dependency to the new task number
4. re render

its not that hard

**Comment by DilipSharma09** [comment]
@mariohmol Does jsgantt-improved support drag-drop task bar in angular

**Comment by mariohmol** [comment]
no... we need to implement... there is some PR that tried to achieve that but we need to work on it

if u can help =)
